workingDataCiteMetadataTemplate-Files.xml

```diff
@@ -1,20 +0,0 @@
-<resource xsi:schemaLocation="http://datacite.org/schema/kernel-3 http://schema.datacite.org/meta/kernel-3/metadata.xsd"
```

Comment:
Why isn't this necessary anymore?

Reply:
Different DataCite schema version maybe (but I'd expect that to be a version change, rather than deletion)?

Reply:
I removed `workingDataCiteMetadataTemplate-Files.xml` because it's not used and is sitting in the root of the repo. The file that actually is being used is `src/main/resources/edu/harvard/iq/dataverse/datacite_metadata_template.xml` but I'm grumpy that it's in the main "dataverse" package, which is a dumping ground, rather than being in a package such as "templates" to help keep the code organized.